Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TestSettings.kt

```diff
@@ -0,0 +1,30 @@
+package de.rki.coronawarnapp.storage
+
+import android.content.Context
+import androidx.core.content.edit
+import de.rki.coronawarnapp.util.CWADebug
+import de.rki.coronawarnapp.util.di.AppContext
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class TestSettings @Inject constructor(
```

Comment:
Class  name could be misleading - I initially thought of Unit tests.
"KeyTestSettings" maybe ?

Reply:
Keeping it only as TestSettings for now, separating them additionally is too much for now, but we could easily refactor this in the future. Because we separated the test settings from the real ones, resetting them will be less problematic.